### DISCUSSION TITLE
Trace/Logging issue

### DISCUSSION THREAD

**Discussion by okbaswc** [microsoft/msquic#1148] [Q&A] [answered]
When I run the command from the docs: 

`netsh.exe trace start overwrite=yes report=dis correlation=dis traceFile=quic.etl provider={ff15e657-4f26-570e-88ab-0796b258d11c} level=0x5 keywords=0xffffffff`

I get an error: **"Invalid value "-outputFormat" for command option "globalKeywords"**

I don't really understand this error, and need some logs to figure out why I'm getting an error in msquic

**Comment by nibanks** [comment, marked as answer]
I bet you are running that in PowerShell. PowerShell is weird with `netsh`. Run it in cmd instead of PowerShell.

**Reply by okbaswc** [reply]
That was it, thanks!
